### ISSUE TITLE
UX improvements

### ISSUE THREAD

**Issue by contrapunctus-1** [genonbeta/TrebleShot#66]
Hi, thanks for putting in your time on this app! FOSS file-sharing apps like TrebleShot are sorely needed, but unfortunately, as it stands, it suffers from poor UX.

The app does not inform the user that...
1. both devices have to be on the same WiFi network (for operation without a hotspot).
2. location mode needs to be 'high accuracy' or 'power saving' to successfully create a hotspot. 

Lastly, setting up TrebleShot seems overly complicated when both users are not on the same WiFi network, which I think is the most common use case. Can anything be done to simplify the process enough such that any non-geek can do it without a helper/documentation? Will #26 improve this?

I tried v1.2.7.6 from F-Droid on a Nokia 6.1 (Android v8.1) and a OnePlus 3T (Android v8)

**Comment by velitasali** [comment]
@contrapunctus-1 
I totally agree. The UX will be improved, but I can't promise for #26, however I can improve setting up hotspot process.

**Comment by chilliger** [comment]
I totally agree. At first I did not even understand (from the app UX) that both devices have to be connected to the same network. Setting up a hotspot was a real hustle. It took serveral attempts and it was really not clear why it suddently started.
